CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 3.2.1 (2022-01-21)
```

Comment:
Included sort of a dummy change so that we can try the new setup (which can be done once we merge this PR; before tags will be ignored since they are in `master`)